src/models/decisions/verification.ts

```diff
@@ -0,0 +1,22 @@
+import { EnumValue } from "../../helpers/advancedTypes";
+import { IDecision, VerificationDecision } from "./decision";
+
+export interface IVerification {
+  confirmed: EnumValue<VerificationDecision>;
+  colorIndex: number;
+  identifier?: symbol;
+}
+
+export class Verification implements IVerification, IDecision {
+  public constructor(data: IVerification) {
+    this.confirmed = data.confirmed;
+    this.colorIndex = data.colorIndex;
+
+    this.identifier = data.identifier ?? Symbol("verification");
+  }
+
+  public type = "verification";
+  public identifier: symbol;
+  public colorIndex: number;
+  public confirmed: EnumValue<VerificationDecision>;
+}
```

Comment:
why is there no tag field here?

Reply:
As per our offline discussion, I've added a `tag` to the abstract `Decision` class (which both `Verification` and `Classification` extend)

This also has the added benefit of simplifying the `addDecision` method on the `Subjects` model, as we no longer have to remove decisions with the same tag

![image](https://github.com/user-attachments/assets/468752f8-8058-4c86-af22-cf417237f435)